### PR TITLE
{bio}[foss/2016a] ALLPATHS-LG v52488

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/a/ALLPATHS-LG/ALLPATHS-LG-52488-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+homepage = 'http://www.broadinstitute.org/software/allpaths-lg/blog/'
+description = """ALLPATHS-LG, the new short read genome assembler."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# source is moved over time, hence multiple URLs below
+source_urls = [
+    'ftp://ftp.broadinstitute.org/pub/crd/ALLPATHS/Release-LG/latest_source_code',
+    'ftp://ftp.broadinstitute.org/pub/crd/ALLPATHS/Release-LG/latest_source_code/2013-06',
+    'ftp://ftp.broadinstitute.org/pub/crd/ALLPATHS/Release-LG/latest_source_code/2013/2013-06',
```
Comment (boegel) [inline easybuild/easyconfigs/a/ALLPATHS-LG/ALLPATHS-LG-52488-foss-2016a.eb]: @verdurin no point in keeping these around, they only apply to older versions?

or at the very least change them to `2015-04` and `2015/2015-04` (even though those directories don't exist yet)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e212a27eb103cd8ca1536e34760a4389 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2448.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/3c8c61fc0633e0bafbcd0a47298b51ce for a full test report.

File: easybuild/easyconfigs/a/ALLPATHS-LG/ALLPATHS-LG-52488-foss-2016a.eb (hunk 1 above)
Comment (verdurin) [inline easybuild/easyconfigs/a/ALLPATHS-LG/ALLPATHS-LG-52488-foss-2016a.eb]: @boegel - yes, I've removed them now, leaving in the note, so they can be added later as required

**Comment by boegel** [comment]
Thanks @verdurin!

**Pull request merged**